### DISCUSSION TITLE
Welcome to packagefactory Discussions!

### DISCUSSION THREAD

**Discussion by aaronparker** [aaronparker/packagefactory#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anotherspot** [comment]
Hi Aaron,

I am hoping to utilise packagefactory in my department to assist with our migration from manual processes in SCCM and some other products over to InTune.

I am having some difficulty in building some App.json Install.json files for some Evergreen apps. (1Password)
Most of the information I can find but am struggling in generation of the PSPackageFactoryGuid

I am also struggling to get the uninstaller to work for LOCALAPPDATA where 1Password is installed to.

Any help / guidance would be fantastic.
